ray-operator/controllers/ray/common/pod.go

```diff
@@ -239,7 +239,7 @@ func DefaultWorkerPodTemplate(ctx context.Context, instance rayv1.RayCluster, wo
 	return podTemplate
 }
 
-func initLivenessAndReadinessProbe(rayContainer *corev1.Container, rayNodeType rayv1.RayNodeType, creatorCRDType utils.CRDType) {
```

Comment:
It is better to avoid having the KubeRay API server call functions within KubeRay. I will only take care of the CRD compatibility, not the internal functions.

Reply:
Oh, here we go again. There is plenty of dependencies already. Do you really assume reproducing all of the code in the API server? This makes no sense to me, sorry. Can we, please, get a second opinion from one of your colleges?
We have been sitting on this PR for several month right now and its getting counter productive. I can do requested enhancements anymore, because we are stuck here for several month